docs/source/rules.rst

```diff
@@ -58,25 +75,24 @@ Parameter Details
 logs
 ~~~~~~~~~~~
 
-``logs`` defines the log sources the rule supports; the ``def`` function block is not run unless this condition is satisfied.
-
-A rule can be run against multiple log sources if desired.
+``logs`` define the log sources the rule supports; the ``def`` function block is not run unless this condition is satisfied.
 
-Log sources (e.g. datasources) are defined in ``conf/sources.json`` and subsequent schemas are defined in ``conf/logs.json``. For more details on how to setup a datasource, please see the Datasources section.
+* A rule can be run against multiple log sources if desired.
+* Log sources (e.g. datasources) are defined in ``conf/sources.json`` and subsequent schemas are defined in ``conf/logs.json``. For more details on how to setup a datasource, please see the Datasources section.
 
 matchers
 ~~~~~~~~
 
-``matchers`` defines the conditions that need to be satisfied in order for the ``def`` function block to run against an incoming record.
+``matchers`` define the additional conditions that must be satisfied for an alert to be triggered.  This serves two purposes:
 
-Matchers are defined in ``rules/matchers.py``
+* To extract common logic from rules, which improves readability and writability
+* To ensure necessary conditions are met in parallel to analysis of an incoming record
 
-Matchers can serve 2 purposes:
+Matchers are generally defined in ``rules/matchers.py``, but they can also in rule files if the following line is added::
```

Comment:
Matchers are defined in ``rules/matchers.py`. If desired, matchers can also be defined in Rule files if the following line is added to the top of the Rule files